locales/en.js

```diff
@@ -0,0 +1,122 @@
+  footerHowGovWorks: 'How government works',
+
+  footerPublicServiceURL:
+    'https://www.canada.ca/en/government/publicservice.html',
+  footerPublicService: 'Public service and military',
+
+  footerGovReportingURL: 'https://www.canada.ca/en/transparency/reporting.html',
+  footerGovReporting: 'Government-wide reporting',
+
+  footerOpenGovURL: 'http://open.canada.ca/en',
+  footerOpenGov: 'Open government',
+
+  aboutgovernmentLink: 'https://www.canada.ca/en/government/system.html',
+  aboutgovernmentText: 'About government',
+
+  // Footnote Links
+
+  footerSocialMediaURL: 'https://www.canada.ca/en/social.html',
+  footerSocialMedia: 'Social media',
+
+  footerMobileAppURL: 'https://www.canada.ca/en/mobile.html',
+  footerMobileApp: 'Mobile applications',
+
+  footerAboutURL: 'https://www.canada.ca/en/government/about.html',
+  footerAbout: 'About Canada.ca',
+
+  footerTermsAndConditionURL:
+    'https://www.canada.ca/en/transparency/terms.html',
+  footerTermsAndCondition: 'Terms and conditions',
+
+  footerPrivacyURL: 'https://www.canada.ca/en/transparency/privacy.html',
+  footerPrivacy: 'Privacy',
+
+  aboutThisSite: 'About this Site',
+  topOfPage: 'Top of Page',
+
+  //footer logo alt text
+  footerCanadaCaAltText: 'Symbol of the Government of Canada',
+
+  //
+  //
+  //  Page Content
+  //
+  //
+
+  //
+  //Report a problem
+  //
+  reportAProblemTitle: 'Report a problem or mistake on this page',
+  reportAProblemButtonString: 'Report a problem on this page',
+  reportAProblemCheckAllThatApply: 'Please select all that apply:',
+  reportAProblemIncorrectInformation: 'Incorrect Information',
+  reportAProblemUnclearInformation: 'Unclear Information',
+  reportAProblemDidNotFindWhatYoureLookingFor:
+    "You didn't find what you were looking for",
+  reportAProblemPageDoesNotWorkWithAdaptiveTechnology:
+    'Page does not work with your adaptive technologies',
+  reportAProblemYoureWorriedAboutYourPrivacy:
+    "You're worried about your privacy",
+  reportAProblemNoWhereElseToGo: "You don't know where else to go for help",
+  reportAProblemOther: 'Other',
+  reportAProblemProvideMoreDetails: 'Provide more details (optional)',
+  reportAProblemNoReply: 'You will not receive a reply. ',
+  reportAProblemEnquiries: 'For enquiries, please contact ',
+  reportAProblemConfidential: 'Your feedback will be confidential',
+  reportAProblemNoPersonalInfo:
+    'Do not include any personal, medical, or financial details, such as your name, social insurance number (SIN), home or business address, phone number or any case or file numbers.',
+  reportAProblemThankYouForYourHelp: 'Thank you for your help!',
+  reportAProblemYouWillNotBeContacted:
+    'You will not receive a reply. For enquiries, please contact us at ',
+  reportAProblemPrivacyStatement: 'Privacy statement',
+  reportAProblemPrivacyStatementLink:
+    'https://www.canada.ca/en/transparency/privacy.html',
+  reportAProblemSubmit: 'Submit',
+}
```

Comment:
If these aren't used can they be removed?

Reply:
yes